src/relative-time.js

```diff
@@ -104,13 +105,13 @@ export default class RelativeTime {
     }
 
     switch(unit) {
-      case "year": return formatters.year(diff.years);
-      case "month": return formatters.month(diff.months);
+      case "year": return this.rtf.format(diff.years, unit);
+      case "month": return this.rtf.format(diff.months, unit);
       // case "week": return formatters.week(diff.weeks);
-      case "day": return formatters.day(diff.days);
-      case "hour": return formatters.hour(diff.hours);
-      case "minute": return formatters.minute(diff.minutes);
-      default: return formatters.second(diff.seconds);
+      case "day": return this.rtf.format(diff.days, unit);
+      case "hour": return this.rtf.format(diff.hours, unit);
+      case "minute": return this.rtf.format(diff.minutes, unit);
+      default: return this.rtf.format(diff.seconds, unit);
```

Comment:
Cool to see `Intl` support is indeed very straightforward.

Now, I would like to see if we can abstract the i18n library. Can we use `format(rtf, value, unit)` where `const format = RelativeTimeFormat.format` that wraps either Intl or globalize (in a plug-n-play way)? We should define how to plug that in. In my code below it was supposed to be overridden by any desired i18n lib, i.e.,  one should override `RelativeTime.initializeFormatters`. Now, I assume we would replace that with two functions: `RelativeTime.initializeFormatter` and `RelativeTime.format`.

Reply:
I'm open for different plug-n-play ideas too.

Reply:
The default wrappers could use `Intl`.

Reply:
@rxaviers  - Would you want to take it over? I'm not sure how you'd like to manage the pluggable backend

Reply:
Let's make relative-time v2 to support `Intl` only? I can keep v1 to support globalize (and potentially a pluggable mechanism). Though, I believe the way to move forward is through standard Intl